src/views/poi_bloc/InternetAccess.jsx

```diff
@@ -0,0 +1,18 @@
+/* global _ */
+import React from 'react';
+
+const InternetAccessBlock = ({
+  block,
+  asString,
+}) => {
+  if (block.wifi === undefined) {
```

Comment:
`if (!block.wifi) {` is safer for this king of test, as it manages `undefined`, `null` and the empty string.